### PR TITLE
Replaced soundcloud player with Spotify player

### LINKED ISSUE #967: Replace Soundcloud player with Spotify player for podcast
In #901 a Soundcloud player was added for the podcast. This player should now be replaced with a Spotify player. The text and link above the player should also be changed or removed completely.

The [Spotify play button](https://developer.spotify.com/documentation/widgets/guides/adding-a-spotify-play-button/) looks like it could be of use. [Here](https://open.spotify.com/show/3jP2S2KKdgBUugBTgoB0B8?si=OXW59TL-SQGLyFWhVGXyWw) is the link to the podcast on Spotify.

### PR DESCRIPTION
**Pull request by Lullebullelukas**
Alexia wrote most of this switch from a soundcloud player to a spotify player but since she quit being a spooder i finished it by just adding some translations.

closes #967

### CODE REVIEW

File: config/locales/onesky_en/index_static_pages.en.yml [hunk 1]
```diff
@@ -1,25 +1,20 @@
...
-# Therefore, you should not modify this file
-# If you want to modify the translation, please do it at OneSky platform
-# If you still want to modify this file directly, please upload this file to OneSky platform after modification in order to update the translation at OneSky
-
 ---
 en:
   static_pages:
     about:
-      description: "F-sektionen arbetar främst med studiebevakning, näringslivskontakt och fritidsnöjen för studenter vid sektionen."
+      description: "The main purpose of the F-guild is to have a say in our education, corporate relations and fun besides schoolwork for the students of the guild."
```
Comment (davidkrantz) [inline config/locales/onesky_en/index_static_pages.en.yml]:
```suggestion
      description: "The F-guild primary works with study surveillance, corporate relations and arranging fun activities for the students of the guild."
```

File: config/locales/onesky_en/index_static_pages.en.yml [hunk 2]
```diff
@@ -1,25 +1,20 @@
...
-      third_paragraph: "As a member of the guild you can choose to be active and committed or to just take part at some events. If you have any questions you can always get in touch with someone from the board or the council of executives, via the Contact tab on the website or in the Board room in the far south of the Hilbert corridor (behind Hilbert Café)! We hope that you will enjoy your time at the F-guild."
-      fourth_paragraph: ""
+      second_paragraph: "The guild has its spaces in the Math building where you can find common rooms, microwave-ovens, and the guild’s own Hilbert Café - where you can buy coffee and great baguettes."
+      third_paragraph: "The F-guild have several councils in charge of different parts of the activities of the guild. Each council is headed by a chairman who is also a part of the guild’s council of executives. The guild also have a board that’s in charge of the central function of the guild."
```
Comment (davidkrantz) [inline config/locales/onesky_en/index_static_pages.en.yml]:
```suggestion
      third_paragraph: "The F-guild has several councils in charge of different parts of the activities of the guild. Each council is headed by a chairman who is also a part of the guild’s council of executives. The guild also has a board that’s in charge of the central function of the guild."
```

File: config/locales/onesky_en/index_static_pages.en.yml [hunk 3]
```diff
@@ -1,25 +1,20 @@
...
+      fourth_paragraph: "As a member of the guild you can choose to be active and committed or take part at some events. We hope that you will enjoy your time in the F-guild. If you have any questions you can always get in touch with someone from the board, via the Contact tab on the website or in Styrelserummet in the far left of the Hilbert corridor (behind Hilbert Café)!"
```
Comment (davidkrantz) [inline config/locales/onesky_en/index_static_pages.en.yml]:
```suggestion
      fourth_paragraph: "As a member of the guild you can choose to be active and committed or take part at some events. We hope that you will enjoy your time in the F-guild. If you have any questions you can always get in touch with someone from the board, via the Contact tab on the website or in Ledningscentralen in the far left of the Hilbert corridor (behind Hilbert Café)!"
```

**Comment by Lullebullelukas** [comment]
I added the requested changes locally and pushed them up so i cannot merge the branch myself

:)
SORRY

**Pull request merged**